src/natupnp_v1.erl

```diff
@@ -114,7 +114,7 @@ get_external_address(#nat_upnp{service_url=Url}) ->
             {Xml, _} = xmerl_scan:string(Body, [{space, normalize}]),
 
             [Infos | _] = xmerl_xpath:string("//s:Envelope/s:Body/"
-                                             "u:GetExternalIPAddressResponse", Xml),
+                                             "*[local-name() = 'GetExternalIPAddressResponse']", Xml),
```

Comment:
Thanks! maybe we could use `[\w]+:` for a generic namespace instead of `u` also. This could make it more generic. I can't try it right now but let me know.

Reply:
it's kind of a pain to test this, but I will try it ASAP

Reply:
```
(miner@127.0.0.1)6> [Infos | _] = xmerl_xpath:string("//s:Envelope/s:Body/[w+]+:GetExternalIPAddressResponse", Xml).
** exception error: no match of right hand side value {error,{1,xmerl_xpath_parse,
                                                              ["syntax error before: ",["'['"]]}}
     in function  xmerl_xpath:string/5 (xmerl_xpath.erl, line 156)
```
I believe what I did is the correct way to wildcard a namespace in XPath 1.0...

Reply:
ok then let's merge it